app/client/components/pages/transferAccountListPage.jsx

```diff
@@ -16,7 +16,7 @@ const mapStateToProps = (state) => {
     transferAccounts: state.transferAccounts,
     mergedTransferAccountUserList: Object.keys(state.transferAccounts.byId)
       .map((id) => {return {...{id, ...state.users.byId[state.transferAccounts.byId[id].primary_user_id]}, ...state.transferAccounts.byId[id]}})
-      .filter(mergedObj => Object.keys(mergedObj).length > 2),  // added to remove {id: 1, balance: 12} objects
+      .filter(mergedObj => Object.keys(mergedObj).length > 3),  // added to remove {id: 1, balance: 12, token: {...}} objects
```

Comment:
feels like there's lot of opportunities for inferring appropriateness just based on length to go wrong - is it possible to explicitly require some attributes exist instead? more fundamentally, this feels like bad data coming from the backend that i should look into...

Reply:
You could add a filter to the transfer account list to check for `transfer_account.users` not being undefined or null... i.e. the transfer account has at least one user associated with it?

Reply:
yeah that sounds good

Reply:
https://github.com/teamsempo/SempoBlockchain/blob/b21953f9b95525b5cacddcfdc8319e030094592c/app/client/components/pages/transferAccountListPage.jsx#L19